v2/delay/delay.go

```diff
@@ -45,6 +46,11 @@ There is some inherent risk of pending function invocations being lost during
 an update that contains large changes. For example, switching from using GOPATH
 to go.mod is a large change that may inadvertently cause file paths to change.
 
+To avoid depending on filepath, users can choose to use a globally unique
```

Comment:
I tried a larger rewrite of  the package and Func docstring. Please see http://go/paste/6371776725516288